CHANGELOG.rst

```diff
@@ -12,18 +12,28 @@ Internal Changes
 
 * **Gateway**: Add endpoint for refreshing expiring Gitlab tokens
 * **Sessions**: Check LFS size and available disk space before cloning
+* **Knowledge Graph**: Improve access tokens management
+* **Knowledge Graph**: Add migration importing Composite Plans into the Triples Store
+* **Knowledge Graph**: Prevent from data corruption caused by duplicate Project creation dates
 
 **Bug fixes**
 
 * **Sessions**: Handle expiring Gitlab tokens
 * **Gateway**: Do not remove Redis clients on logout
 * **Gateway**: Address security vulnerabilities
+* **Knowledge Graph**: Fix vulnerabilities in Graph Services' docker images
+* **Knowledge Graph**: Handle cases when Plan Invalidation Time is wrong
+* **Knowledge Graph**: Made Cross-Entity Search API to sort in a case-insensitive way
+* **Knowledge Graph**: Treat query phrase given to the Cross-Entity Search API as a String
```

Comment:
What does this mean?

Reply:
Yes, this is a tricky one. What about this:
```
Escape Lucene keywords from Cross-Entity Search `query` parameter
```

Reply:
I prefer your proposal - gives me some feeling for what it is...